examples/pinot_quickstart_timeout.py

```diff
@@ -0,0 +1,46 @@
+from pinotdb import connect
+import time
+import pytest
+import httpx
+
+## Start Pinot Quickstart Batch
+## docker run --name pinot-quickstart -p 2123:2123 -p 9000:9000 -p 8000:8000 \
+##    -d apachepinot/pinot:latest QuickStart -type hybrid
+
+def run_pinot_quickstart_timeout_example() -> None:
+
+    #Test 1 : Try without timeout and wait for more than 5 seconds. The query should execute without exceptions.
+
+    conn = connect(host="localhost", port=8000, path="/query/sql", scheme="http")
+    curs = conn.cursor()
+    sql = "SELECT * FROM airlineStats LIMIT 5"
+    print(f"Sending SQL to Pinot: {sql}")
+    time.sleep(5.001)
+    curs.execute(sql)
```

Comment:
the `curs.execute(sql)` was called after `time.sleep(5.001)` was done? this doesn't seem to be useful to validate the query didn't timeout

Reply:
i might be wrong but is the timeout start counting on `conn.cursor()` or start on `curs.execute(sql)`?

Reply:
Ah, the sleep is not necessary. I have removed them. Thank you for pointing this out. 

I was under the impression it starts from httpx.client(). It seems to start from receiving one chunk of the data to the other , so starting from `curs.execute`: `The read timeout specifies the maximum duration to wait for a chunk of data to be received (for example, a chunk of the response body). If HTTPX is unable to receive data within this time frame, a ReadTimeout exception is raised.`[documented here](https://www.python-httpx.org/advanced/)